plugins/modules/ios_service.py

```diff
@@ -0,0 +1,689 @@
+  cisco.ios.ios_service:
+    running_config: "{{ lookup('file', 'parsed.cfg') }}"
+    state: parsed
+
+# Task Output
+# -----------
+#
+# parsed:
+#   counters: 5
+#   dhcp: true
+#   password_encryption: true
+#   password_recovery: true
+#   prompt: true
+#   slave_log: true
+#   tcp_keepalives_in: true
+#   tcp_keepalives_out: true
+#   timestamps:
+#   - datetime_options:
+#       localtime: true
+#       msec: true
+#       show_timezone: true
+#       year: true
+#     msg: debug
+#     timestamp: datetime
+#   - datetime_options:
+#       localtime: true
+#       msec: true
+#       show_timezone: true
+#       year: true
+#     msg: log
+#     timestamp: datetime
+"""
+
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+after:
+  description: The resulting configuration after module execution.
+  returned: when changed
+  type: dict
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+commands:
+  description: The set of commands pushed to the remote device.
+  returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
+  type: list
+  sample:
+    - sample command 1
+    - sample command 2
+    - sample command 3
+rendered:
+  description: The provided configuration in the task rendered in device-native format (offline).
+  returned: when I(state) is C(rendered)
+  type: list
+  sample:
+    - sample command 1
+    - sample command 2
+    - sample command 3
+gathered:
+  description: Facts about the network resource gathered from the remote device as structured data.
+  returned: when I(state) is C(gathered)
+  type: list
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+parsed:
+  description: The device native config provided in I(running_config) option parsed into structured data as per module argspec.
+  returned: when I(state) is C(parsed)
+  type: list
+  sample: >
+    This output will always be in the same format as the
+    module argspec.
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.argspec.service.service import (
+    ServiceArgs,
+)
+from ansible_collections.cisco.ios.plugins.module_utils.network.ios.config.service.service import (
+    Service,
+)
+
+
+def main():
+    """
+    Main entry point for module execution
+
+    :returns: the result form module invocation
+    """
+    module = AnsibleModule(
+        argument_spec=ServiceArgs.argument_spec,
+        mutually_exclusive=[["config", "running_config"]],
+        required_if=[
+            ["state", "merged", ["config"]],
+            ["state", "replaced", ["config"]],
+            ["state", "overridden", ["config"]],
```

Comment:
overridden is not a supported state right?

Reply:
Oups... 😅